### PR TITLE
Safari support with clipboard api

### PR DESCRIPTION
**Pull request by hobbitronics**
1. - Copy was not working in Safari so used the clipboard API
2. - Chrome was not accepting the clipboard item so created a new Blob
3. - Added an alert if copy fails

### CODE REVIEW

File: src/components/App.svelte [hunk 1]
```diff
@@ -41,18 +41,15 @@ $: rememberInfo && localStorage.setItem('skypeName', skypeName)
...
+    const data = [new ClipboardItem({ "text/html": sig })]
 
-  function handleCopy (event) {
-    event.clipboardData.setData('text/html', sig)
-    event.clipboardData.setData('text/plain', sig)
-    
-    event.preventDefault()
+    await navigator.clipboard.write(data)
+  } catch {
+    alert('Failed to copy you signature.')
```
Comment (Schparky) [inline src/components/App.svelte:52]:
```suggestion
    alert('Failed to copy your signature.')
```

**Review by Schparky — APPROVED** [review]
Quick turnaround on the bug report!

**Comment by fillup** [comment]
Do we not need the `plain/text` content as well?

**Comment by hobbitronics** [comment]
> Do we not need the `plain/text` content as well?

This seems to catch it all. Chrome doesn't support multiple clipboardItems with this api, which is supposed to supersede the old method, so a little disappointing.

**Comment by merchako** [comment]
Thanks for putting up with Safari's quirks, Mick. You're a trooper!

**Pull request merged**